documentation/2-options.md

```diff
@@ -936,9 +936,14 @@ Optionally overrides the value of [`--max-http-header-size`](https://nodejs.org/
 **Type: `boolean`**\
 **Default: `false`**
 
-By default, requests will not use [method rewriting](https://datatracker.ietf.org/doc/html/rfc7231#section-6.4).
+Specifies if the HTTP request method should be [rewritten as `GET`](https://tools.ietf.org/html/rfc7231#section-6.4) on redirects.
 
-For example, when sending a `POST` request and receiving a `302`, it will resend the body to the new location using the same HTTP method (`POST` in this case). To rewrite the request as `GET`, set this option to `true`.
+As the [specification](https://tools.ietf.org/html/rfc7231#section-6.4) prefers to rewrite the HTTP method only on `303` responses, this is Got's default behavior.
+
+However, since the spec also allows to rewrite to `GET` for `301` and `302` responses and many user agents - including, e.g., [`curl`](https://everything.curl.dev/http/redirects#get-or-post) and all major browsers - do so by default, this option may be used to mimic the same behavior and reach compatibility with servers expecting it.
```

Comment:
```suggestion
It's also possible to do method rewriting for `301` and `302` responses, as allowed by the spec. This is the behavior followed by `curl` and browsers.
```

Reply:
done, with another formulation which seemed even better and terser to me.